### ISSUE TITLE
Civic logo doesn't look good in Windows High-Contrast mode

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#1049]
## What
When Windows high-contrast mode is enabled, the Civic logo in the site's header isn't looking so good:

![screenshot of the Civic logo. The inner background is black, which makes the C in the center, and half of the logo's border, hard to see.](https://user-images.githubusercontent.com/169471/65368654-f88eef00-dbf8-11e9-99f5-a02ef2aee572.png)

The SVG needs updating to add a white internal background to make the C in the center stand out.
